Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -130,73 +117,63 @@ protected bool DrawBacktrackProfileButton(string message, UnityEngine.Object act
-                result = renderButton();
-            GUILayout.EndHorizontal();
-            return result;
-        }
-
         /// <summary>
         /// Helper function to render header correctly for all profiles
         /// </summary>
         /// <param name="title">Title of profile</param>
         /// <param name="description">profile tooltip describing purpose</param>
+        /// <param name="isProfileInitialized">profile properties are full initialized for rendering</param>
         /// <param name="backText">Text for back button if not rendering as sub-profile</param>
         /// <param name="backProfile">Target profile to return to if not rendering as sub-profile</param>
-        /// <returns>true to render rest of profile as-is or false if profile/MRTK is in a state to not render rest of profile contents</returns>
-        protected bool RenderProfileHeader(string title, string description, BackProfileType returnProfileTarget = BackProfileType.Configuration)
+        protected void RenderProfileHeader(string title, string description, bool isProfileInitialized = true, BackProfileType returnProfileTarget = BackProfileType.Configuration)
         {
-            RenderMixedRealityToolkitLogo();
+            RenderMRTKLogo();
 
-            if (!MixedRealityInspectorUtility.CheckMixedRealityConfigured())
+            var profile = target as BaseMixedRealityProfile;
+            if (!RenderAsSubProfile && !profile.IsCustomProfile)
             {
-                return false;
+                EditorGUILayout.HelpBox("Default Mixed Reality Toolkit profiles cannot be edited", MessageType.Warning);
+                // TODO: Place clone button. MixedRealityProfileCloneWindow requires parent profile though which is not known here
+                //EditorGUILayout.HelpBox("Default MRTK profiles cannot be edited. Create a clone of this profile to modify settings.", MessageType.Warning);
             }
-
-            if (DrawBacktrackProfileButton(returnProfileTarget))
+            else
             {
-                return false;
+                if (!isProfileInitialized && profile.IsCustomProfile)
+                {
+                    EditorGUILayout.HelpBox("Some properties may not be editable in this profile. Please refer to the error messages below to resolve editing.", MessageType.Warning);
+                }
+
+                MixedRealityInspectorUtility.CheckMixedRealityConfigured();
+
+                if (MixedRealityToolkit.IsInitialized)
+                {
+                    if (IsProfileInActiveInstance())
+                    {
+                        DrawBacktrackProfileButton(returnProfileTarget);
+                    }
+                    else if (!isProfileInitialized)
+                    {
+                        EditorGUILayout.HelpBox("This profile is not assigned to the active MRTK instance in this scene. Some properties may not be editable", MessageType.Error);
+                    }
```

Comment:
I assume we don't include a button to add the instance here because we're waiting on #4435 to be merged.